src/main/webapp/css/profileBuilder.css

```diff
@@ -0,0 +1,19 @@
+
+.button_slide {
```

Comment:
Assuming what's being served on appspot is up to date (as it seems to be), this renders as a grey button with a maroon border (box-shadow) with white text. Is that the intended style? The white text on grey background is hard to read.

Reply:
+1 regarding making the buttons easier to read

Reply:
I see your concern, in the frontend, the buttons show up as a dark grey background, with white text and maroon border. It's pretty readable on my end.

[Image Link](https://screenshot.googleplex.com/HpBuhg0tu1R.pngl)

Reply:
Done

Reply:
Chatted offline, looks like the version on appspot wasn't up to date + didn't see the "Front-end appearence" link in the original post; Looks good